### ISSUE TITLE
xmrig freezes at uncertain times

### ISSUE THREAD

**Issue by MTCHANNELL** [RainbowMiner/RainbowMiner#2628]
![image](https://github.com/RainbowMiner/RainbowMiner/assets/94814602/32429a78-d7f0-4db5-a559-fe782911af60)

![image](https://github.com/RainbowMiner/RainbowMiner/assets/94814602/41d7e79e-a380-4210-8168-efc79bd7d000)

![image](https://github.com/RainbowMiner/RainbowMiner/assets/94814602/d19530d0-028d-464f-8f78-b673e83c3218)

![image](https://github.com/RainbowMiner/RainbowMiner/assets/94814602/873cabf1-fc66-4686-9f59-2efab5857760)

When I mine with Xmrig, when I look at the webgui, Xmrig freezes at undetermined times. It doesn't get better again. However, if I notice it and turn it off and on completely, it gets fixed. I am using the latest version of xmrig. It appears to be mining in the task manager, but when I look at the statistics of the pool, there are no workers there, and when I look at the "rainbowminer" webgui, I see that it is frozen.

**Comment by MTCHANNELL** [comment]
algorithm : ghostrider
pool : zpool
I don't know if xmrig causes problems with different algorithms. I only do ghostrider mining.

**Comment by RainbowMiner** [comment]
Yes, ghostrider is a bit different. How many cores/threads have you assigned to CPU mining?

**Comment by MTCHANNELL** [comment]
I don't usually meddle in Miner's business. I leave it automatic :) but I would also like to point out this. If I mine Ghostrider with cpuminerrplant, there is no problem. Note: If I use cpuminerrplant, it causes "ms" increase problems after a certain period of time. So I need to use xmrig. I only get the freezing issue in xmrig. I think I know what you mean. You can say that it freezes because it uses all the cores, but if I mine using xmrig with a manual bat file, there is no problem. However if I scrape Ghostrider with xmrig into Rainbowminer the xmrig freezes anytime within 24 hours. However, a computer can be used. I can navigate between interfaces with webgui. It's a strange situation. PC does not freeze. Miner freezes and gets stuck. Unfortunately, I cannot use your "rainbowminer" application right now just because of this problem:( I could always control it with webgui. I liked it very much. I hope we can solve this problem.
